onnxruntime/contrib_ops/cpu/bert/attention_cpu_base.h

```diff
@@ -145,10 +145,23 @@ class AttentionCPUBase : public AttentionBase {
       const int loop_len = batch_size * num_heads_;
       const float alpha = scale_ == 0.0f ? 1.0f / sqrt(static_cast<float>(head_size)) : scale_;
 
-      // The cost of Gemm
-      const double cost = static_cast<double>(head_size) * sequence_length * total_sequence_length;
+      TensorOpCost unit_cost;
+      unit_cost.compute_cycles = static_cast<double>(2 * sequence_length * head_size * total_sequence_length);
+      unit_cost.bytes_loaded = double(sequence_length * head_size * sizeof(T) + head_size * total_sequence_length * sizeof(T));
+      unit_cost.bytes_stored = double(sequence_length * total_sequence_length * sizeof(T));
```

Comment:
nit: may define a temp variable (like intermediate_chunk_bytes) for `sequence_length * total_sequence_length * sizeof(T)` since it is used in multiple places in function.